MIP40/mip40.md

```diff
@@ -38,43 +38,43 @@ Gives an overview of the most critical characteristics of Budgets.
 Describes the components of a Core Unit Budget proposal.
 
 **MIP40c3: Adding/Modifying Core Unit Budget Process**
-Describes the process for adding a budget to a new Core Unit or modifying the Core Unit's existing budget.
+Describes the process for adding a Budget to a new Core Unit or modifying the Core Unit's existing Budget.
 
 **MIP40c4: Budget Implementation Details**
 Provides clarity on how the budget payouts are addressed from a practical perspective once approved.
 
 ## Motivation
 
-The Budget Framework MIP allows Governance to manage and prioritize Core Units in a standardized way, promoting transparency and reducing Governance overhead and minimizing abuse and increase fairness in the budget process.
-The budget system gives Facilitators the flexibility to modify their plans away from the initially submitted budget breakdown if conditions were to change. If they need to allocate the money differently, this would be possible but never exceeding the aggregate amount defined by their Budget Implementation code.
-The Framework can help Governance guide a given Core Unit, as they could vote (or not) to attach multiple budgets to a Core Unit, each with a different, more specific objective.
+The Budget Framework MIP allows Governance to manage and prioritize Core Units in a standardized waythat promotes transparency and fairness. Minimzing overhead and potental for abuse in the Budget process.
```

Comment:
Think these changes may be boarding a bit more on the subjective edits side. I appreciate the clean up effort, but we also want to respect the author's autonomy